client/router.js

```diff
@@ -36,12 +37,20 @@ Router.prototype.route = function(path, options) {
   };
 
   this._routes.push(route);
+  if (options.name) {
+    this._pathsMap[options.name] = path;
```

Comment:
Yeah, we can but I can't find use case that accessing to route object to by name for now. So I prefer minimal approach.

When we really need it, it's easy to change.